test/unit/model_runners/extractors/test_jumpstart_extractor.py

```diff
@@ -234,6 +234,27 @@ def test_extractor_when_model_spec_is_missing_inference_configs(self, sagemaker_
                 sagemaker_session=sagemaker_session,
             )
 
+    @patch("sagemaker.session.Session")
+    def test_extractor_when_default_payloads_found_outside_inference_configs(self, sagemaker_session):
+        """
+        GIVEN the default payloads data that is found is empty.
+        WHEN a JumpStartExtractor is being initialized.
+        THEN the correct exception is raised.
```

Comment:
nit:please change this description since there is no exeception raised for this case.

Reply:
Oh good catch. I forgot to change the description. Will change this.